libflux/core/src/parser/mod.rs

```diff
@@ -50,7 +51,7 @@ impl Parser {
             s,
             t: None,
             errs: Vec::new(),
-            blocks: HashMap::new(),
+            blocks: HashMap::default(),
```

Comment:
Can you help me connect how calling `default()` here ends up using the DefaultHasher defined above?

Reply:
I'm not entirely sure. I was getting build failures with `new`, and [the FnvHasher documentation](https://doc.servo.org/fnv/index.html) uses `default`. That worked, so I went with it.